src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -5,7 +5,8 @@
     <template v-if="!isEditing">
       <v-layout row id="summary-registered-address" class="mx-0">
         <v-flex xs3>
-          <label>Registered Office</label>
+          <label v-if="isChangeFiling">Business Addresses</label>
+          <label v-else>Registered Office</label>
```

Comment:
I'm thinking this should be a resource.

Reply:
For sure, can do.
Seemed like a pretty small thing, so i wasn't sure at the time.

Reply:
Updated 👍